import_3dm.py

```diff
@@ -1,3 +1,15 @@
+bl_info = {
+    "name": "Import Rhinoceros 3D",
+    "author": "jesterKing",
```

Comment:
You can add your own name as well:

```suggestion
    "author": "jesterKing, romio82",
```

Reply:
I'd rather add my name as soon as I contribute something substantial to the code.. ;) I'll have some time over the weekend. Maybe I'll get some small addition ready that would be worth mentioning my name..